index.html

```diff
@@ -199,80 +214,298 @@ <h2>Multi-Post <br />Stories</h2>
       </container>
     </div>
 
-    <div class="about">
-      <h2 class="about-title">About Myself</h2>
-      <p class="about-description">
-        Hello I’m a software developer! I can help you build a product , feature
-        or website. Look through some of my work and experience! If you like
-        what you see and have a project you need coded, don’t hesitate to
-        contact me.
-      </p>
-
-      <div class="media">
-        <h3>LET'S CONNECT</h3>
-        <div class="socials">
-          <ul>
-            <li class="icon">
-              <a href="https://github.com/peter-mwovi/"
-                ><img src="./images/github.png" alt="github link"
-              /></a>
+    <div class="grid-1b">
+      <div class="rows">
+        <div class="photo1">
+          <img class="work1b-img" alt="work 1" src="images/image4.png" />
+        </div>
+        <div class="work1b">
+          <h2 class="work-text1b poppins">Tonic</h2>
+          <ul class="flex1">
+            <li>
+              <p class="canopy poppins">CANOPY</p>
+            </li>
+            <li>
+              <img class="dot" alt="dot" src="images/dot1.png" />
+            </li>
+            <li>
+              <p class="bed poppins">Back End Dev</p>
             </li>
-            <li class="icon">
-              <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
-                ><img src="./images/Linkedin_icon.png" alt="linked link"
-              /></a>
+            <li>
+              <p>
+                <img class="dot" alt="dot" src="images/dot1.png" />
+              </p>
+            </li>
+            <li>
+              <p class="year poppins">2015</p>
             </li>
-            <li class="icon">
-              <a href="#"><img src="./images/doits.png" alt="doit" /></a>
+          </ul>
+          <p class="desc2 poppins">
+            A daily selection of privately personalized reads; no accounts or
```

Comment:
- [optional] just like you did for the headline and about section, kindly make your cards responsive till they reach the desktop version. This will make your work perfect in all screen sizes.
![Screenshot (1131)_LI](https://github.com/peter-mwovi/portfolio_setup/assets/65298048/d7d40477-b9c7-42d1-9543-4b72dd33cb76)